app/domain/logs.rb

```diff
@@ -34,6 +34,15 @@ module Conjur
       code: "CONJ00042D"
     )
 
+    ValidatingProxyList = ::Util::TrackableLogMessageClass.new(
+      msg:  "Validating proxy list {0-proxy-list}",
```

Comment:
@shulifink can you review please?

Reply:
Is this the same as the trusted proxy list?
If so, do we want "Validating trusted proxy list {0-proxy-list}"?